.github/workflows/playwright.yaml

```diff
@@ -0,0 +1,40 @@
+name: 'playwright-test-report'
+on:
+  pull_request:
+    workflow_dispatch:
+jobs:
+  test-report:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Run Playwright tests
+      run: npm run test:coverage
+      env:
+        CI: true
+
+    - name: Code Coverage Summary Report
+      uses: irongut/CodeCoverageSummary@v1.3.0
+      with:
+        filename: 'coverage/cobertura-coverage.xml'
```

Comment:
This is a file generated by `Run Playwright tests` step?

Reply:
Correct, it's a coverage report from nyc. We run text, cobertura and lcov reports for different needs, specified in .nycrc